File: python/KimElektro.py
# Kelas KimElektro untuk merepresentasikan produk elektronik
class KimElektro:
    # =============================
    # Constructor: inisialisasi atribut
    # =============================
    def __init__(self, _id, _nama, _JenisAlat, _harga):
        self.id = _id          # ID unik produk
        self.nama = _nama      # Nama produk
        self.JenisAlat = _JenisAlat  # JenisAlat produk
        self.harga = _harga    # Harga produk (integer)

    # =============================
    # Getter: mengambil nilai atribut
    # =============================
    def getID(self):
        return self.id
    def getNama(self):
        return self.nama
    def getJenisAlat(self):
        return self.JenisAlat
    def getHarga(self):
        return self.harga

    # =============================
    # Setter: mengubah nilai atribut
    # =============================
    def setID(self, _id):
        self.id = _id
    def setNama(self, _nama):
        self.nama = _nama
    def setJenisAlat(self, _JenisAlat):
        self.JenisAlat = _JenisAlat
    def setHarga(self, _harga):
        self.harga = _harga

    # =============================
    # Method untuk menampilkan info barang
    # =============================
    def printElektro(self):
        print(f"ID      : {self.id}")
        print(f"Nama    : {self.nama}")
        print(f"JenisAlat: {self.JenisAlat}")
        print(f"Harga   : {self.harga}")

File: java/KimElektro.java
// Kelas KimElektro merepresentasikan produk elektronik
public class KimElektro {
    // =============================
    // Atribut kelas (private untuk encapsulation)
    // =============================
    private String id;        // ID unik setiap produk
    private String nama;      // Nama produk
    private String JenisAlat;  // JenisAlat produk (misal: TV, Radio, dll)
    private int harga;        // Harga produk dalam bentuk integer

    // =============================
    // Constructor: digunakan untuk membuat objek KimElektro baru
    // =============================
    public KimElektro(String _id, String _nama, String _JenisAlat, int _harga) {
        id = _id;
        nama = _nama;
        JenisAlat = _JenisAlat;
        harga = _harga;
    }

    // =============================
    // Getter: mengambil nilai atribut
    // =============================
    public String getID() { return id; }
    public String getNama() { return nama; }
    public String getJenisAlat() { return JenisAlat; }
    public int getHarga() { return harga; }

    // =============================
    // Setter: mengubah nilai atribut
    // =============================
    public void setID(String _id) { id = _id; }
    public void setNama(String _nama) { nama = _nama; }
    public void setJenisAlat(String _JenisAlat) { JenisAlat = _JenisAlat; }
    public void setHarga(int _harga) { harga = _harga; }

    // =============================
    // Method untuk menampilkan info barang ke console
    // =============================
    public void printElektro() {
        System.out.println("ID      : " + id);
        System.out.println("Nama    : " + nama);
        System.out.println("JenisAlat: " + JenisAlat);
        System.out.println("Harga   : " + harga);
    }
}